qml/pages/ChatSelectionPage.qml

```diff
@@ -61,8 +67,53 @@ Dialog {
 
         model: chatListModel
         delegate: ChatListViewItem {
-            ownUserId: overviewPage.ownUserId
+            ownUserId: chatSelectionPage.myUserId
+            Loader { // checking permissions takes a while, so we defer those calculations
+                id: visibleLoader
+                asynchronous: true
+                sourceComponent: Component {
+                    QtObject {
+                        property var chatGroupInformation: ({})
+                        property bool visible: false
+                        Component.onCompleted: {
+                            if(chatSelectionPage.state === "forwardMessages") {
+                                var chatType = display.type['@type'];
+                                if(chatType === "chatTypePrivate" || chatType === "chatTypeSecret") {
+                                    visible = true
+                                    return;
+                                }
+                                else if (chatType === "chatTypeBasicGroup" ) {
+                                    chatGroupInformation = tdLibWrapper.getBasicGroup(display.type.basic_group_id);
+                                }
+                                else if (chatType === "chatTypeSupergroup" ) {
+                                    chatGroupInformation = tdLibWrapper.getSuperGroup(display.type.supergroup_id);
+                                }
+
+                                visible = (chatGroupInformation.status["@type"] === "chatMemberStatusCreator"
+                                        || chatGroupInformation.status["@type"] === "chatMemberStatusAdministrator"
+                                        || (chatGroupInformation.status["@type"] === "chatMemberStatusMember"
```

Comment:
but not always :/

Reply:
In this case, I can actually omit the property completely (isn't used outside the function at all) and just use a JS var. 
But there's another bug hidden in there – thanks for making me aware.